presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -175,4 +188,21 @@ private void execute(String sql)
             statement.execute(sql);
         }
     }
+
+    private QueryRunner createQueryRunnerWithLegacyCoercion()
+            throws Exception
+    {
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put("deprecated.legacy-char-to-varchar-coercion", "true")
+                .build();
+
+        Map<String, String> databaseProperties = ImmutableMap.<String, String>builder()
+                .put("connection-url", mysqlServer.getJdbcUrl())
+                .build();
+
+        QueryRunner queryRunner = new DistributedQueryRunner(getSession(), 3, properties);
+        queryRunner.installPlugin(new MySqlPlugin());
+        queryRunner.createCatalog("mysql", "mysql", databaseProperties);
```

Comment:
technically, if we throw on this line, the `queryRunner` won't get closed.

since this method is used in single place, i would inline it. This will take the test code slightly more verbose, but will make the resource management more reliable. The test code would look like this:

```
try (QueryRunner queryRunner = new DistributedQueryRunner(getSession(), 3, ImmutableMap.of("deprecated.legacy-char-to-varchar-coercion", "true")) {
  queryRunner.installPlugin(new MySqlPlugin());
  queryRunner.createCatalog("mysql", "mysql", ImmutableMap.of("connection-url", mysqlServer.getJdbcUrl()));

  ... tests using queryRunner
}
```

Reply:
Gotcha, didn't realize this was following a disposal pattern.  Done.